blocks/text/text.css

```diff
@@ -13,6 +13,10 @@ main .block.text.narrow {
   margin-bottom: 3.5rem;
 }
 
+main .block.text.text-right > * {
```

Comment:
This would only right align the immediate children. I hope that's what you want.

Reply:
Yes